orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/SomaticFindingsChapter.java

```diff
@@ -156,10 +156,10 @@ private void addKataegisPlot(@NotNull Document document)
 
     private void addSomaticAmpDels(@NotNull Document document)
     {
-        String driverAmpsDelsTitle = "Driver amps/dels";
-        String nearDriverGainsTitle = "Potentially interesting near-driver amps";
-        String suspectGainsTitle = "Other regions with amps";
-        String suspectLossesTitle = "Regions with deletions in genes in other autosomal regions";
+        String driverAmpsDelsTitle = "Driver amplifications/homologous deletions";
```

Comment:
It is a "homozygous" deletion, not a "homologous" deletion! 

Also I would just use " and" rather than "/"